[2025-06-27T04:25:54.566+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-06-27T04:25:54.592+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ga4_raw_data_elt_v1.0.start_task scheduled__2025-03-20T05:00:00+00:00 [queued]>
[2025-06-27T04:25:54.601+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ga4_raw_data_elt_v1.0.start_task scheduled__2025-03-20T05:00:00+00:00 [queued]>
[2025-06-27T04:25:54.602+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-06-27T04:25:54.729+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): start_task> on 2025-03-20 05:00:00+00:00
[2025-06-27T04:25:54.778+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1461) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-06-27T04:25:54.794+0000] {standard_task_runner.py:72} INFO - Started process 1493 to run task
[2025-06-27T04:25:54.861+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'ga4_raw_data_elt_v1.0', 'start_task', 'scheduled__2025-03-20T05:00:00+00:00', '--job-id', '361', '--raw', '--subdir', 'DAGS_FOLDER/GA4_data_elt.py', '--cfg-path', '/tmp/tmpa9nsa6x1']
[2025-06-27T04:25:54.888+0000] {standard_task_runner.py:105} INFO - Job 361: Subtask start_task
[2025-06-27T04:25:55.073+0000] {task_command.py:467} INFO - Running <TaskInstance: ga4_raw_data_elt_v1.0.start_task scheduled__2025-03-20T05:00:00+00:00 [running]> on host f79cc72a155d
[2025-06-27T04:25:55.399+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ga4_raw_data_elt_v1.0' AIRFLOW_CTX_TASK_ID='start_task' AIRFLOW_CTX_EXECUTION_DATE='2025-03-20T05:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-20T05:00:00+00:00'
[2025-06-27T04:25:55.414+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-06-27T04:25:55.443+0000] {logging_mixin.py:190} INFO - start
[2025-06-27T04:25:55.450+0000] {logging_mixin.py:190} INFO - ds: 2025-03-20
[2025-06-27T04:25:55.451+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-06-27T04:25:55.502+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-06-27T04:25:55.507+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=ga4_raw_data_elt_v1.0, task_id=start_task, run_id=scheduled__2025-03-20T05:00:00+00:00, execution_date=20250320T050000, start_date=20250627T042554, end_date=20250627T042555
[2025-06-27T04:25:55.544+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-06-27T04:25:55.627+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-06-27T04:25:55.638+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
